### PR TITLE
Update Rubocop

### PR DESCRIPTION
**Pull request by bf4**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -59,6 +64,18 @@ Style/BlockDelimiters:
...
+  EnforcedStyle: semantic
+
+Style/TrailingCommaInLiteral:
+  EnforcedStyleForMultiline: consistent_comma
+
+Style/ConditionalAssignment:
+  Enabled: false
+
+Style/DotPosition:
+  EnforcedStyle: trailing
```
Comment (bf4) [inline .rubocop.yml]: @rails-api/ams Are we dot-trailing or dot-leading people?

Comment (remear) [inline .rubocop.yml]: These days I prefer leading.

**Comment by remear** [comment]
What remains to be done on this?

**Comment by bf4** [comment]
@remear 

> What remains to be done on this?

I honestly don't recall. But it would need to be regenerated off of current master. Feel free to re-use, close, push to, whatever. Probably easier to start over but see the config changes for reference.

You can see my steps in the commits.

**Comment by NullVoxPopuli** [comment]
ref: https://github.com/rails-api/active_model_serializers/pull/1560

**Comment by groyoh** [comment]
Closing in favor of #1560

**Pull request closed without merging**